src/debug.ts

```diff
@@ -0,0 +1,5 @@
+import debug from 'debug';
+
+export function createDebug(name: string): debug.Debugger {
+    return debug(`@map-colonies/config:${name}`);
+}
```

Comment:
I'm not too fond of it being in the main files hierarchy

Reply:
moved to util